compose.yml

```diff
@@ -1,13 +1,27 @@
 services:
   data_service:
+    depends_on:
+      database:
+        condition: service_healthy
     build: data_service
-    entrypoint: cargo watch -x run
+    command: sh -c "
+      diesel setup &&
+      diesel migration generate --diff-schema create_match_data &&
```

Comment:
Moved migration steps here so we can diff against current schema and fix database if we need to

Reply:
Quick flag that in times of trouble we can check our migration manually by commenting out the migration run here and then inspecting the migrations file manually to see if it makes sense.